tests/system/action/meeting/test_delete.py

```diff
@@ -47,6 +49,7 @@ def test_delete_permissions_can_manage_committee(self) -> None:
         self.assert_status_code(response, 200)
         self.assert_model_deleted("meeting/1")
 
+    @pytest.mark.skip()
```

Comment:
Here are two options (just for remembering):

1. Delete this line, when the example-data in the main-repo (OpenSlides/OpenSlides) is up-to-date
2. Extract this test into an own and mark it as `optional`, so a PR can be merged, even when that test fails

Reply:
I'm strongly against merging PRs with failing tests, that is the whole meaning why we do the tests.